### PR TITLE
Add Fingerprint 2.0 for 2020 Toyota Corolla Cross (Taiwanese Version)

### PR DESCRIPTION
**Pull request by protonchang**
* Add fingerprint for engine, eps, esp, fwdCamera.
* This car does not support Stop & Go, also no Full-Speed DRCC
* No ALC and ACC below 18mph (30kph)

* Dongle ID: 94545a9472a6c942

### CODE REVIEW

File: selfdrive/car/toyota/values.py [hunk 1]
```diff
@@ -649,12 +650,14 @@ class CAR:
       b'\x018965B12530\x00\x00\x00\x00\x00\x00',
       b'\x018965B1255000\x00\x00\x00\x00',
       b'8965B12361\x00\x00\x00\x00\x00\x00',
+      b'8965B16011\x00\x00\x00\x00\x00\x00'
```
Comment (VirtuallyChris) [inline selfdrive/car/toyota/values.py:653]: need a , after this line

**Comment by protonchang** [comment]
Also, I'm not sure if I should update README.md or not, as Corolla Cross in the US market has Full-Speed DRCC and SnG feature, but it's not public available yet.
I suspect it will have a  different fingerprint compare to the Taiwanese version.

**Comment by pd0wm** [comment]
Does openpilot stop steering under 18mph?

**Comment by protonchang** [comment]
Yes, complete disengadge below 18mph

**Pull request merged**